### PR TITLE
feat: activate CDN using env

### PR DESCRIPTION
**Pull request by jmfrancois**
**What is the problem this PR is trying to solve?**

* Current CDN activattion need rebuild.
* initiator duplicate parts of a code that is already available for onprem

**What is the chosen solution to this problem?**

* share talendAddScripts and talendAddStyles method
* remove --env.cdn option support
* activate CDN if value of TALEND_INITIATOR_URL is not its placeholder value '@@INITIATOR_URL@@';

This PR follow https://github.com/Talend/cdn-content/pull/13

**Please check if the PR fulfills these requirements**

* [x] The PR commit message follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md)
* [ ] Tests for the changes have been added (for bug fixes / features) And [non reg](./screenshots.md) done before need review
* [ ] Docs have been added / updated (for bug fixes / features)
* [ ] Related design / discussions / pages (not in jira), if any, are all linked or available in the PR

**[ ] This PR introduces a breaking change**

### CODE REVIEW

File: packages/config-react-webpack/src/inject.js [hunk 1]
```diff
@@ -1,28 +1,45 @@
...
+		const style = document.createElement('link');
+		style.setAttribute('rel', 'stylesheet');
+		style.setAttribute('media', 'print');
+		style.setAttribute('onload', 'this.media="all"');
+		style.setAttribute('href', finalUrl);
+		document.head.insertBefore(style, title);
+	});
+}
+window.talendAddStyles = talendAddStyles;
+window.talendAddScripts = talendAddScripts;
```
Comment (frassinier) [inline packages/config-react-webpack/src/inject.js:31]: why putting them outside the IIFE if you attach them to the window?

File: packages/config-react-webpack/src/inject.js [hunk 2]
```diff
@@ -1,28 +1,45 @@
...
-			style.setAttribute('onload', 'this.media="all"');
-			style.setAttribute('href', url);
-			document.head.insertBefore(style, title);
-		});
-	}
 	document.addEventListener('DOMContentLoaded', function onDocumentReady() {
-		addStyles(cssFiles);
-		addScripts(jsFiles);
+		if (window.TALEND_INITIATOR_URL !== '@@INITIATOR_URL@@') {
+			talendAddScripts([window.TALEND_INITIATOR_URL]);
```
Comment (frassinier) [inline packages/config-react-webpack/src/inject.js:38]: I don't get it? what is `[window.TALEND_INITIATOR_URL]` supposed to do?

Comment (jmfrancois) [inline packages/config-react-webpack/src/inject.js:38]: because of that we have to remove the event listener on the cdn initiator itself

Comment (jmfrancois) [reply packages/config-react-webpack/src/inject.js:38]: this is the url of the initiator

**Review by frassinier — APPROVED** [review]

**Pull request merged**